### ISSUE TITLE
 yaml: control characters are not allowed

### ISSUE THREAD

**Issue by kronphe** [kubernetes/minikube#17390]
### What Happened?

minikube update-context

### Attach the log file

 C:\Users\rckon\AppData\Local\Temp\minikube_update-context_455f11127792dd8a866b496362cf59c36c3e255a_0.log

### Operating System

None

### Driver

None

**Comment by spowelljr** [comment]
Hi @kronphe, could you include the command you were trying to run and the full output.

**Comment by kronphe** [comment]
Hi,
Thanks for your quick reaction.  While installing minikube on my windows 11
macchine, I was having some misconfiguration. So I ran `kubectl logs
dashboard-metrics-scraper-64c9f7bf9d-4mwsh -n kubernetes-dashboard`.
I succeed with the troubleshooting and everything is running good now.

I just had to delete and start minikube another time and it worked!.

Thanks once more.

RK


Le mar. 10 oct. 2023 à 17:36, Steven Powell ***@***.***> a
écrit :
